tests/ambsym/mresolution_identifier_1.nim

```diff
@@ -0,0 +1,9 @@
+import mresolution_identifier_2
```

Comment:
The changes to the `ambsym` category are the biggest i've made so far - i'm getting more confident in these changes. 
So please, take a more careful look at these.